event-streams/grafana-dashboards/ibm-eventstreams-kafka.json

```diff
@@ -299,116 +329,6 @@
       "title": "Filter by Broker: $kafka_broker",
       "type": "row"
     },
-    {
-      "cacheTimeout": null,
-      "colorBackground": false,
-      "colorPostfix": false,
-      "colorPrefix": false,
-      "colorValue": false,
-      "colors": [
-        "#299c46",
-        "rgba(237, 129, 40, 0.89)",
-        "#d44a3a"
-      ],
-      "datasource": "prometheus",
-      "format": "none",
-      "gauge": {
-        "maxValue": 100,
-        "minValue": 0,
-        "show": false,
-        "thresholdLabels": false,
-        "thresholdMarkers": true
-      },
-      "gridPos": {
-        "h": 4,
-        "w": 4,
-        "x": 0,
-        "y": 6
-      },
-      "id": 109,
-      "interval": null,
-      "links": [],
-      "mappingType": 1,
-      "mappingTypes": [
-        {
-          "name": "value to text",
-          "value": 1
-        },
-        {
-          "name": "range to text",
-          "value": 2
-        }
-      ],
-      "maxDataPoints": 100,
-      "nullPointMode": "connected",
-      "nullText": null,
-      "options": {},
-      "postfix": "",
-      "postfixFontSize": "50%",
-      "prefix": "",
-      "prefixFontSize": "50%",
-      "rangeMaps": [
-        {
-          "from": "null",
-          "text": "N/A",
-          "to": "null"
-        }
-      ],
-      "sparkline": {
-        "fillColor": "rgba(31, 118, 189, 0.18)",
-        "full": false,
-        "lineColor": "rgb(31, 120, 193)",
-        "show": false,
-        "ymax": null,
-        "ymin": null
-      },
-      "tableColumn": "",
-      "targets": [
-        {
-          "expr": "scalar(kube_pod_status_phase{pod=\"$kafka_broker\", phase=\"Failed\"}*1) +  scalar(kube_pod_status_phase{pod=\"$kafka_broker\", phase=\"Pending\"}*2) + scalar(kube_pod_status_phase{pod=\"$kafka_broker\", phase=\"Running\"}*3) +  scalar(kube_pod_status_phase{pod=\"$kafka_broker\", phase=\"Succeeded\"}*4) +scalar(kube_pod_status_phase{pod=\"$kafka_broker\", phase=\"Unknown\"}*5)",
-          "refId": "A"
-        }
-      ],
-      "thresholds": "",
-      "timeFrom": null,
-      "timeShift": null,
-      "title": "Pod Phase",
```

Comment:
This panel doesn't seem to work and because it's conveying the same info as the `Number of brokers online` panel maybe we should remove.